code/fcn_2D_segm/data_loader.py

```diff
@@ -0,0 +1,231 @@
+import os
```

Comment:
No, I left the two data_loader.py from unet and fcn_2D_segm just in case. But they can be removed